Encrypt_decrypt_text_in_images/Readme.md

```diff
@@ -0,0 +1,7 @@
+<h1>How to use</h1>
+<br>
```

Comment:
Please try to make the `README` more descriptive.

Reply:
Done